cloudformation/membership-attribute-service.json

```diff
@@ -177,6 +177,16 @@
                 "Resource": "arn:aws:dynamodb:*:*:table/friendly-tailor-PROD"
               },
               {
+                "Effect":"Allow",
+                "Action":["sns:Publish"],
+                "Resource":{"Ref":"TopicGiraffe"}
+              },
+              {
+                "Effect":"Allow",
+                "Action":["sns:Publish"],
+                "Resource":{"Ref":"TopicGiraffeUAT"}
```

Comment:
@philwills Agreed.  Unfortunately due to the membership architecture, the production instance needs to populate the UAT Topic for test users otherwise we pollute the production data with test users.